mimic-iv/concepts/organfailure/kdigo_stages.sql

```diff
@@ -3,35 +3,35 @@
 -- Baseline creatinine is defined as the lowest creatinine in the past 7 days.
 
 -- get creatinine stages
-with cr_stg AS
+WITH cr_stg AS
 (
   SELECT
     cr.stay_id
     , cr.charttime
     , cr.creat_low_past_7day 
     , cr.creat_low_past_48hr
     , cr.creat
-    , case
+    , CASE
         -- 3x baseline
-        when cr.creat >= (cr.creat_low_past_7day*3.0) then 3
+        WHEN cr.creat >= (cr.creat_low_past_7day*3.0) THEN 3
         -- *OR* cr >= 4.0 with associated increase
-        when cr.creat >= 4
+        WHEN cr.creat >= 4
         -- For patients reaching Stage 3 by SCr >4.0 mg/dl
         -- require that the patient first achieve ... acute increase >= 0.3 within 48 hr
         -- *or* an increase of >= 1.5 times baseline
-        and (cr.creat_low_past_48hr <= 3.7 OR cr.creat >= (1.5*cr.creat_low_past_7day))
-            then 3 
+        AND (cr.creat_low_past_48hr <= 3.7 OR cr.creat >= (1.5*cr.creat_low_past_7day))
+            THEN 3 
         -- TODO: initiation of RRT
-        when cr.creat >= (cr.creat_low_past_7day*2.0) then 2
-        when cr.creat >= (cr.creat_low_past_48hr+0.3) then 1
-        when cr.creat >= (cr.creat_low_past_7day*1.5) then 1
-    else 0 end as aki_stage_creat
-  FROM `physionet-data.mimiciv_derived.kdigo_creatinine` cr
+        WHEN cr.creat >= (cr.creat_low_past_7day*2.0) THEN 2
+        WHEN cr.creat >= (cr.creat_low_past_48hr+0.3) THEN 1
+        WHEN cr.creat >= (cr.creat_low_past_7day*1.5) THEN 1
+    ELSE 0 END AS aki_stage_creat
+  FROM `physionet-data.mimiciv_derived.kdigo_creatinine`
```

Comment:
accidentally dropped the `cr` alias here, which breaks the query

Reply:
Sorry for the rookie mistake :D fixed!